Can't build docker image from latest master

Hello there , 

### How did you install OpenDroneMap? (Docker, natively, ...)?
Docker
### What's your browser and operating system? (Copy/paste the output of https://www.whatismybrowser.com/)
N/A
### What is the problem?
Can't build docker image from latest master
### What should be the expected behavior? If this is a feature request, please describe in detail the changes you think should be made to the code, citing files and lines where changes should be made, if possible.
Building image without issue
### How can we reproduce this? (What steps did you do to trigger the problem? What parameters are you using for processing? If possible please include a copy of your dataset uploaded on Google Drive or Dropbox. Be detailed)
```
git clone https://github.com/OpenDroneMap/ODM.git odm
cd odm
docker build -t rmitrev/odm -f ./Dockerfile . > out.txt 2>&1
```

Will attach building logs later today.

error is the same as in your Github Actions builds

```
exit code 2
```

[build log](https://drive.google.com/file/d/1QDrs-GYcjyRMa6z7WV_oVgzMnQYNhVFx/view?usp=sharing)

some error near the end
```
...
 [0m [91mmake[5]: *** [libs/MVS/CMakeFiles/MVS.dir/build.make:158: libs/MVS/CMakeFiles/MVS.dir/Mesh.cpp.o] Error 1
......
 [91mmake[4]: *** [CMakeFiles/Makefile2:491: libs/MVS/CMakeFiles/MVS.dir/all] Error 2
 [0m [91mmake[3]: *** [Makefile:149: all] Error 2
 [0m [91mmake[2]: *** [CMakeFiles/openmvs.dir/build.make:135: openmvs/stamp/openmvs-build] Error 2
 [0m [91mmake[1]: *** [CMakeFiles/Makefile2:328: CMakeFiles/openmvs.dir/all] Error 2
 [0m [91mmake[1]: *** Waiting for unfinished jobs
....
```
Since 3 day's all workflows are broken

--- a/SuperBuild/cmake/External-OpenMVS.cmake
+++ b/SuperBuild/cmake/External-OpenMVS.cmake
@@ -2,8 +2,8 @@ set(_proj_name openmvs)
 set(_SB_BINARY_DIR "${SB_BINARY_DIR}/${_proj_name}")
 
 externalproject_add(vcg
-    GIT_REPOSITORY  https://github.com/cdcseacave/VCG.git
-    GIT_TAG         master
+    GIT_REPOSITORY  https://github.com/OpenDroneMap/VCG.git
+    GIT_TAG         280
     UPDATE_COMMAND  ""
     SOURCE_DIR      ${SB_SOURCE_DIR}/vcg
     CONFIGURE_COMMAND ""

